index.html

```diff
@@ -63,86 +70,100 @@ <h1>Eat Donuts in the Wilderness</h1>
         </div>
       </div>
     </section>
-    <section class="container-size">
-      <div class="card-group">
-        <div class="card">
-          <img src="./assets/images/ocean.jpg" class="card-image" alt="...">
-          <div class="card-main">
-            <h5 class="card-heading">Donuts by the Sea</h5>
-            <p class="card-body">Lots and lots of text about how great donuts are when you eat them by the sea. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut congue a quam in gravida. Ut scelerisque at est consectetur tincidunt. Sed volutpat massa non arcu imperdiet consectetur.</p>
+    <section class="container">
+      <div class="row justify-content-between">
```

Comment:
This is a great solution! The bootstrap pre-built options would be to use either the `card-group` or how they do the card grid (but your solution is close to theirs on the grid - I think the main difference is how theirs will break down on a smaller screen). Here's a link to that part of the cards: https://getbootstrap.com/docs/5.1/components/card/#card-layout